src/LinuxTracing/TracerImpl.cpp

```diff
@@ -718,6 +734,38 @@ static std::vector<ThreadName> RetrieveInitialThreadNamesSystemWide(uint64_t ini
   return thread_names;
 }
 
+// Returns a vector of pairs mapping the tid in the target process namespace to the corresponding
+// tid in the root namespace.
+static std::vector<std::pair<pid_t, pid_t>> RetrieveInitialTidMapping(pid_t target_pid) {
+  std::vector<std::pair<pid_t, pid_t>> tid_mapping;
```

Comment:
Why are we not already returning a map instead of translating the vector to a map later?